core-bundle/src/Resources/contao/templates/forms/form_captcha.html5

```diff
@@ -34,7 +34,13 @@
         e.required = false;
         p.style.display = 'none';
 
-        setTimeout(function() { e.value = '<?= $this->getSum() ?>'; }, 5000);
+        setTimeout(() => {
+          fetch(<?= json_encode(Contao\System::getContainer()->get('router')->generate('contao_frontend_captcha', ['_locale' => Contao\System::getContainer()->get('request_stack')->getCurrentRequest()->getLocale()])) ?>).then((r) => r.json()).then((d) => {
```

Comment:
Can we move this logic to the `FormCaptcha` class and only pass the generated route to the template?

Reply:
Of course 🙈

Done in e1cea31ca5b0c306a9ed45bb844960bd5eea0807